Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -268,10 +274,19 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
 {
   const bool enable_wii_tools = !emulation_started || !SConfig::GetInstance().bWii;
   m_perform_online_update_menu->setEnabled(enable_wii_tools);
+
+  IOS::HLE::Kernel ios;
```

Comment:
Be careful with this. The NAND should only be accessed when Wii emulation is not running (i.e. when `enable_wii_tools` is true); otherwise it will be accessed both by this code and by the emulated software at the same time, which may cause inconsistent data to be returned and possibly even data corruption.